Gemfile

```diff
@@ -56,7 +56,7 @@ gem "manageiq-ssh-util",              "~>0.1.1",       :require => false
 gem "memoist",                        "~>0.15.0",      :require => false
 gem "mime-types",                     "~>3.0",         :path => File.expand_path("mime-types-redirector", __dir__)
 gem "money",                          "~>6.13.5",      :require => false
-gem "more_core_extensions",           "~>3.7"
+gem "more_core_extensions",           "~>4.0"
```

Comment:
I think we should drop the version requirement here since `more_core_extensions` is very closely tied to `manageiq-gems-pending`.  Most of it has been extracted from `manageiq-gems-pending` so the minimum version should be set there.  Maybe even add a comment that the version should not be set here.

@Fryguy ^ sound reasonable?

Reply:
At that point, this wouldn't need to be marked as WIP

Reply:
Yeah that makes sense to me